### DISCUSSION TITLE
Filament Excel Export

### DISCUSSION THREAD

**Discussion by NekoMeowCat** [pxlrbt/filament-excel#195] [Q&A]
Im using a filament excel export package by Dennis Koch (pxlrbt\FilamentExcel\Actions\Tables\ExportBulkAction). I tried integrating 
->askForFileName() on the BulkExportAction of the package but obviously it doesnt work can anybody give me some ideas how I can work around it?

**Comment by NekoMeowCat** [comment]
![90](https://github.com/user-attachments/assets/8cbfe683-8c1d-4392-b15d-9bbfd8cf72cc)

**Comment by pxlrbt** [comment]
> ->askForFileName() on the BulkExportAction 

Please check the docs. The method shouldn't be called on the Action, but on the Export class
